bokeh/embed/util.py

```diff
@@ -239,6 +252,9 @@ def __eq__(self, other: Any) -> bool:
         else:
             return self.elementid == other.elementid
 
+    def __repr__(self) -> str:
+        return f"RenderRoot(elementid={self.elementid}, id={self.id}, name={self.name}, tags={self.tags})"
+
```

Comment:
It would be better to rewrite this class as a dataclass, and remove all of this code.

Reply:
It would be better, but this PR still improvement, since it resolves the broken docs (and those docs will still be correct even with a future rewrite). I will make a separate issue about applying dataclasses more generally.